redesign.php

```diff
@@ -13,6 +13,11 @@
             <!-- Wrapper for slides -->
             <div class="carousel-inner" role="listbox">
                 <div class="item active" style="background: url('assets/img/home/carousel/bloom-poster.jpg');">
+                    <div class="info">
+                        <img src="assets/img/home/carousel/bloom-logo.png" alt="Field Day - Learning Games" />
```

Comment:
Since these are all hardcoded, I think it would be good to include the title for each game in the `alt` text.